### ISSUE TITLE
Speed and acceleration prediction

### ISSUE THREAD

**Issue by Eirenliel** [SlimeVR/SlimeVR-OpenVR-Driver#11]
We can use accelerometer to get better results of it, but actually it isn't even needed. SteamVR needs to know the speed of the trackers to predict movement for future frames, we can calculate it from the past movements, or from accelerometer. We can also try asking BNO085 to predict future in a few ms. Either way, this can be useful.

**Comment by ButterscotchV** [comment]
Seems to be solved by #64

**Comment by kitlith** [comment]
there may still be some server-side changes to make? not sure.

**Comment by ButterscotchV** [comment]
> there may still be some server-side changes to make? not sure.

Yes, there's still https://github.com/SlimeVR/SlimeVR-Server/issues/25 which we can use to track this, as the driver side is done.
